onnxruntime/core/session/inference_session.h

```diff
@@ -438,13 +509,14 @@ class InferenceSession {
 #ifdef ENABLE_LANGUAGE_INTEROP_OPS
   InterOpDomains interop_domains_;
 #endif
-  // used to support platform telemetry 
-  static std::atomic<uint32_t> global_session_id_;  // a monotonically increasing session id
-  uint32_t session_id_;                             // the current session's id
-  uint32_t total_runs_since_last_;                  // the total number of Run() calls since the last report
-  long long total_run_duration_since_last_;         // the total duration (us) of Run() calls since the last report
-  TimePoint time_sent_last_;                        // the TimePoint of the last report
-  const long long kDurationBetweenSending = 1000* 1000 * 60 * 10;  // duration in (us).  send a report every 10 mins
+
+  // used to support platform telemetry
+  static std::atomic<uint32_t> global_session_id_;                  // a monotonically increasing session id
+  uint32_t session_id_;                                             // the current session's id
+  uint32_t total_runs_since_last_;                                  // the total number of Run() calls since the last report
+  long long total_run_duration_since_last_;                         // the total duration (us) of Run() calls since the last report
+  TimePoint time_sent_last_;                                        // the TimePoint of the last report
+  const long long kDurationBetweenSending = 1000 * 1000 * 60 * 10;  // duration in (us).  send a report every 10 mins
```

Comment:
Should we package this into a struct? This looks wordy here.

Reply:
AFAIK - this hasn't changed in this PR. Should we hold off on such refactoring later in the interest of making it easy to review ?

Reply:
+1 to Dmitri's comment. Can be refactored in a separate PR though.

Reply:
If its okay, I prefer doing this in a separate PR. Nothing has changed in these lines and this kind of refactoring are best split out into separate changes if they don't align with the theme of this PR.